docs/LLS_Metallicities-Wotta/mcmc_met.py

```diff
@@ -0,0 +1,258 @@
+    ## Run the MCMC
+    ########################
+    
+    ##Should we optimize?
+    print 'Ready to run mcmc for {}'.format(sy)
+    if optim in ['guess', 'guess_NHI']:
+        obsinfo['met'] = float(met_guess) ##make sure it's a float
+        obsinfo['dens'] = float(dens_guess) ##make sure it's a float
+        obsinfo['carbalpha'] = float(carbalpha_guess) ##make sure it's a float
+    else:
+        optim = False
+    
+    logUconstraint = str(logUconstraint) ##convert to string
+    nthread = int(nthread) ##make sure it's an integer
+    
+    ##Run
+    mcmc=mcmc_ions(observ, obsinfo, grid_fil, 
+                   logUconstraint=logUconstraint, UVB=UVB,
+                   nwalkers=(nwalkers),
+                   nsamp=(nsteps), optim=optim, threads=nthread,
+                   outsave=outsave, testing=testing)
+    
+    print('All done!!')
+
+
+
+################################################
+################################################
+
+
+def read_input_file(guesses_file, row_index_to_run):
+
+    ##Define the columns in the input file
+    ok_to_run_in = []
+    sightline_in = []
+    met_guess_in = []
+    dens_guess_in = []
+    carbalpha_guess_in = []
+    carbalpha_use_in = []
+    logUconstraint_use_in = []
+    uvb_in = []
+    notes_in = []
+
+    ##Read the input file
+    infile=open(guesses_file, 'r')
+    for line in infile:
+        if line and not line.startswith("#"):
+            linespl = line.strip().split()
+            ##
+            ok_to_run_in.append(linespl[0])
+            sightline_in.append(linespl[1])
+            met_guess_in.append(linespl[2])
+            dens_guess_in.append(linespl[3])
+            carbalpha_guess_in.append(linespl[4])
+            carbalpha_use_in.append(linespl[5])
+            logUconstraint_use_in.append(linespl[6])
+            uvb_in.append(linespl[7])
+            notes_in.append(linespl[8])
+
+
+    ##We only care about the row that is given as the command line argument
+    ok_to_run = ok_to_run_in[row_index_to_run]
+    sightline = sightline_in[row_index_to_run]
+    met_guess = met_guess_in[row_index_to_run]
+    dens_guess = dens_guess_in[row_index_to_run]
+    carbalpha_guess = carbalpha_guess_in[row_index_to_run]
+    carbalpha_use = carbalpha_use_in[row_index_to_run]
+    logUconstraint_use = logUconstraint_use_in[row_index_to_run]
+    uvb = uvb_in[row_index_to_run]
+    notes = notes_in[row_index_to_run]
+
+
+
+    return ok_to_run, \
+        sightline, \
+        met_guess, \
+        dens_guess, \
+        carbalpha_guess, \
+        carbalpha_use, \
+        logUconstraint_use, \
+        uvb, \
+        notes
+
+
+
+################################################
+################################################
+################################################
+
+
+
+def main(row_to_run):
+    
+    ##row_to_run should be indexed from 1 not 0!! We subtract 1 below!
+    
+    ##CBW adjust
+    guesses_file = "MCMC_initial_guesses-run_me.dat"

```

Comment:
Done.  I'll have to update the docs once I get everything sorted out.  But how to call `pyigm_mtlmcmc.py` with the `--wotta` option is listed via a comment at the bottom of that file.  It'll get to the docs eventually.  I need to make sure things are working first.